packages/osl-management-console-image/resources/modules/osl/management-console/module.yaml

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+schema_version: 1
+name: com.redhat.osl.management.console
+version: "0.0.0"
+description: "OpenShift Serverless Management Console Image Configuration"
+
+execute:
+  - script: configure-osl.sh
+  - script: configure-httpd.sh
+
+envs:
+  - name: SONATAFLOW_MANAGEMENT_CONSOLE_PORT
+    value: 8080
+
+artifacts:
+  - name: sonataflow-management-console-webapp
+    description: "## UPDATED DURING BUILD ##"
+    dest: /home/kogito/management-console/app
+    target: sonataflow-management-console-webapp-image-build.zip
+    md5: "## UPDATED DURING BUILD ##"
+  - name: image-env-to-json-standalone
```

Comment:
For those 2 artifacts (image-env-to-json-standalone and EnvJson.schema.json) we need to copy them to the image outside this `artifacts` of cekit, otherwise cekit will try to use lookaside cache.